routes/web.php

```diff
@@ -17,6 +17,8 @@
     return view('welcome');
 });
 
+Route::resource('post', PostController::class);
```

Comment:
You referenced `Auth::id()` in your `store()` method, so this route should use the `auth` middleware to ensure it is only accessible to an authenticated user.